auth_password_settings/README.rst

```diff
@@ -0,0 +1,22 @@
+Enforce users to set password more complex
+=========================================
+This module will allow adminstrator to set password determination rules.
+To do this, User need to go Settings / Configuration / General Configuration
+General Configuration will have Password Settings section to set determination rules.
+-- Minimum Password length in character
+-- Use Capital Letter
+-- Use digit
+-- Use special character
+
+Contributors
+------------
+* Dhaval Patel
+
+History:
+2015-02-24: 1.0
+==================
+
+Initial Development 
+-----------------
+    * Develop password setting module to enforce user to set their passowrd more complex
+
```

Comment:
please add the OCA boilerplate text. 

You can use https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst as inspiration